### ISSUE TITLE
Update Trezor library

### ISSUE THREAD

**Issue by spylogsster** [brave/brave-browser#20640]
There is audit issue with `node-fetch`
```
└─┬ trezor-connect@8.2.1-extended
  ├─┬ @trezor/rollout@1.2.1
  │ └─┬ cross-fetch@3.1.4
  │   └── node-fetch@2.6.1
  ├── node-fetch@2.6.7
  └─┬ trezor-link@1.7.3
    └── node-fetch@2.6.7 deduped
```

https://github.com/advisories/GHSA-r683-j2x4-v87g

**Comment by spylogsster** [comment]
Waiting for this fix https://github.com/lquixada/cross-fetch/issues/118

**Comment by imsys** [comment]
@spylogsster cross-fetch 3.1.5 already fixes it by using node-fetch 2.6.7, that is a backport of the patch. https://github.com/node-fetch/node-fetch/commit/1ef4b560a17e644a02a3bfdea7631ffeee578b35
Just the warning was not fixed for node-fetch 2.6.7 yet.

**Comment by imsys** [comment]
Edit: the warning is already fixed, and should show `cross-fetch 3.1.5` and  `node-fetch 2.6.7` as having the patched version.

**Comment by Daniel4SE** [comment]
Can you fix the issue of #20634, I also meet that problem

**Comment by srirambv** [comment]
<table>
   <tr>
      <th>Brave</th>
      <th colspan="3">1.36.90 Chromium: 98.0.4758.87 (Official Build) beta (64-bit)</th>
   </tr>
   <tr>
      <th>Revision</th>
      <th colspan="3"><code>e4cd00f135fb4d8edc64c8aa6ecbe7cc79ebb3b2-refs/branch-heads/4758@{#1002}</code></th>
   </tr>
   <tr>
      <th>OS
      </th>
      <th>:ballot_box_with_check: Linux</th>
      <th>:ballot_box_with_check: Windows 11 Version Dev
         <br/> (Build 22523.1000)
      </th>
      <th>:ballot_box_with_check: macOS Version 12.0.1
         <br/> (Build 21C52)
      </th>
   </tr>
   <tr>
      <td colspan=4></td>
   <tr>
      <td colspan=4>
         <ul>
         <li>Verified steps from <a href=https://github.com/brave/brave-core/pull/11960>brave/brave-core#11960</a></li>
         <li>Verified able to import Trezor accounts</li>
         <li>Encountered <a href=https://github.com/brave/brave-browser/issues/21041>#21041</a></li>
        </ul>
      </td>
   </tr>
</table>
